packages/babel-traverse/src/path/modification.ts

```diff
@@ -399,14 +399,17 @@ export function pushContainer<
   return path.replaceWithMultiple(verifiedNodes);
 }
 
-/**
- * Hoist the current node to the highest scope possible and return a UID
- * referencing it.
- */
-export function hoist<T extends t.Node>(
-  this: NodePath<T>,
-  scope: Scope = this.scope,
-) {
-  const hoister = new PathHoister<T>(this, scope);
-  return hoister.run();
+if (!process.env.BABEL_8_BREAKING && !USE_ESM) {
+  /**
+   * Hoist the current node to the highest scope possible and return a UID
+   * referencing it.
+   */
+  // eslint-disable-next-line no-restricted-globals
+  exports.hoist = function hoist<T extends t.Node>(
```

Comment:
Out of curiosity, what was this ever used for?

Reply:
It seems to be used by `babel-plugin-transform-react-constant-elements`, I'm not sure, it has no use now.